src/main/java/com/conveyal/gtfs/validator/PatternFinderValidator.java

```diff
@@ -172,8 +185,7 @@ public void complete(ValidationResult validationResult) {
                             lastValidDeparture = prevDeparture;
                         }
                     }
-
-                    if (stopById.containsKey(stopOrLocationId)) {
+                    if (stopById.containsKey(stopOrLocationIdOrLocationGroupId)) {
```

Comment:
Is the below the implementation for `setStatementParameters` for `PatternStop`? Same question for the other blocks that call `addLocationOrLocationGroup`.

Reply:
The first block covers pattern stops. The other blocks cover locations and location groups. Because locations and location groups are very similar, they are covered in the same method (addLocationOrLocationGroup). What is your concern with this section of code? I'm not sure I follow.

Reply:
Is `setStatementParameters` intented to contain that code for the respective pattern classes?

Reply:
Yes, although a couple of params were not in the correct order. Thanks for the heads up, sort of :-)

Reply:
ah right sorry! Yep, refactored.